packages/core/.storybook/main.js

```diff
@@ -2,8 +2,11 @@ const { heftStoryGlobs, storykitAddons, features, framework, core } = require('@
 
 module.exports = {
   stories: heftStoryGlobs,
-  addons: storykitAddons,
-  features,
+  addons: [...storykitAddons, '@storybook/addon-interactions'],
+  features: {
+    ...features,
+    interactionsDebugger: true,
+  },
```

Comment:
do we want to commit this change?

Reply:
I believe so! This configurations change allows us to perform interaction testing through storybook, which is particularly useful for animation testing!